tests/test_authorisation.py

```diff
@@ -0,0 +1,171 @@
+from config import Config
+from tests.conftest import create_valid_token
+from tests.conftest import test_assessor_claims
+from tests.conftest import test_commenter_claims
+from tests.conftest import test_lead_assessor_claims
+from tests.conftest import test_roleless_user_claims
+
+
+class TestAuthorisation:
+    def test_any_unauthorised_route_redirects_to_home(self, flask_test_client):
+        """
+        GIVEN an unauthorised user
+        WHEN the user tries to access any route
+        THEN tne user is redirected to the "/" root
```

Comment:
lovely tests 👍  👏 
typo here on 'tne'

Reply:
Ah great spot! That's about a 1.5 pixel difference between n and h!